### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by maosan132**
Hello Reviewers, Please give my code a shot and give me suggestions for improvement, I would appreciate it so much. It encompasses milestone 5 of the Bookstore project.

- [ ] The full design specs in this Zeplin project. 
- [ ] Implement the styling of the different components based on the design.

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -21,30 +21,26 @@ const BookList = () => {
...
-      </table>
+    <div className="BooksList">
+      <header className="flex">
+        <div className="header-leftside flex">
+          <h1>Bookstore CMS</h1>
+          <span>BOOKS</span>
+          <CategoryFilter filter={handleCategoryFilter} />
+        </div>
+        {/* <img src={userIcon} alt="user-icon" /> */}
+      </header>
```
Comment (rukundoeric) [inline src/containers/BooksList.js]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/styles/App.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+.BooksForm select {
+  width: 20%;
+}
+
+.BooksForm button {
+  width: 15%;
+  background-color: var(--azure-color);
+  color: var(--white-color);
+}
```
Comment (rukundoeric) [inline src/styles/App.css:68]: - [ ] <b>REQUIRED:</b> Your app doesn't look exactly like mockup design

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_9_9_2021_1-22-23 AM |
   - The a circle progress bar is missing on each of your book item. 
   -  Around each of your Action buttons, there are some dots which are not present in the mockup design, kindly remove them.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi guys, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] The live-demo in your README is for a different project. kindly replace it with the correct one.
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/containers/BooksList.js (hunk 1 above)
Comment (maosan132) [reply src/containers/BooksList.js]: DOne, thanks

**Review by rukundoeric — APPROVED** [review]
Hi @maosan132,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**